common/util/rviz_plugins/autoware_perception_rviz_plugin/README.md

```diff
@@ -0,0 +1,52 @@
+# autoware_perception_rviz_plugin
+
+## Purpose
+
+This plugin is used to generate dummy pedestrians, cars, and obstacles in planning simulator.
+
+## Overview
+
+The CarInitialPoseTool sends a topic for generating dummy car.  
+The PedestrianInitialPoseTool sends a topic for generating dummy pedestrian.  
+The UnknownInitialPoseTool sends a topic for generating dummy obstacle.  
```

Comment:
```suggestion
The CarInitialPoseTool sends a topic for generating a dummy car.  
The PedestrianInitialPoseTool sends a topic for generating a dummy pedestrian.  
The UnknownInitialPoseTool sends a topic for generating a dummy obstacle.  
```
Nit: Article `a` might be needed.

Reply:
fix https://github.com/tier4/AutowareArchitectureProposal.iv/pull/591/commits/5048af02e1379a1139e991530bf0fc60c661ba37